lib/features/manage_account/domain/model/create_new_rule_filter_request.dart

```diff
@@ -0,0 +1,13 @@
+
+import 'package:equatable/equatable.dart';
+import 'package:rule_filter/rule_filter/tmail_rule.dart';
+
+class CreateNewRuleFilterRequest with EquatableMixin {
+
+  final List<TMailRule> newListRules;
```

Comment:
please separate `current list` and new item will be added

Reply:
Why do that?

Reply:
Ok, to make it clear we will do so. I have updated it.